README.md

```diff
@@ -9,4 +9,19 @@ git submodule update --init --recursive
 cd bls
 make -f Makefile.onelib ETH_CFLAGS=-DBLS_ETH LIB_DIR=lib
 zig build
-```
\ No newline at end of file
+```
+
+## Test
+
+first you should build bls as described above
+
+when you run `zig build test`, the spec tests are not run by default.
+
+if you want to run spec tests, you need to download the test vectors and add `-Dspec=true` to the zig build command.
```

Comment:
it would be better to describe the detail how to download the test vectors or just a command line here